src/lv_core/lv_obj.h

```diff
@@ -263,6 +266,12 @@ typedef struct
                                                ... [x]: "lv_obj" */
 } lv_obj_type_t;
 
+typedef struct _lv_hit_test_info_t
```

Comment:
Can it go to `lv_obj.c`?

Reply:
Objects need to be able to access the structure, so it has to be visible to them in a header file.

Reply:
Oh, true!